tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -77,12 +77,17 @@ internal static partial class ConfigurationKeys
         public const string AppSecExtraHeaders = "DD_APPSEC_EXTRA_HEADERS";
 
         /// <summary>
-        /// Specifies if the AppSec traces should be explicitly kept or droped.
-        /// Default is true, to keep all traces, false will drop all traces.
+        /// Specifies if the AppSec traces should be explicitly kept or dropped.
+        /// Default is true, to keep all traces, false means drop all traces (by setting AutoReject as sampling priority).
         /// For internal testing only.
         /// </summary>
         internal const string AppSecKeepTraces = "DD_APPSEC_KEEP_TRACES";
 
+        /// <summary>
+        /// Limits the amount of AppSec traces sent per second with an integer value, strictly positive.
+        /// </summary>
+        internal const string AppSecTraceRateLimit = "DD_APPSEC_TRACE_RATE_LIMIT";
```

Comment:
Clearly out of scope of this PR, but we have started to use partial classes for confKeys. We could move all the appsec ones to a new partial class for readability

Reply:
makes total sense, given the timing 😅 will keep in mind for next PRs!